lib/ruby_lsp/ruby_lsp_rails/support/callbacks.rb

```diff
@@ -16,6 +16,10 @@ module Callbacks
             "around_create",
             "after_create",
             "after_commit",
+            "after_create_commit", # alias for after_commit
+            "after_update_commit", # alias for after_commit
+            "after_destroy_commit", # alias for after_commit
+            "after_save_commit", # alias for after_commit
```

Comment:
Nit: no need for these comments tbh
```suggestion
            "after_create_commit",
            "after_update_commit",
            "after_destroy_commit",
            "after_save_commit",
```